### PR TITLE
[ekf2][sdlog] Add more ekf2 debugging information and magneto setting and add sdlogger filename

### PR DESCRIPTION
**Pull request by fvantienen**
This needs https://github.com/paparazzi/pprzlink/pull/105

### CODE REVIEW

**Comment by gautierhattenberger** [comment]
pprzlink updated

**Comment by fvantienen** [comment]
Then this should be able to be merged if approved. After that I will rebase https://github.com/paparazzi/paparazzi/pull/2502 which will then successfully compile.

**Comment by fvantienen** [comment]
Wait I need to update the reference to pprzlink as well I see now. But this would require both the #2502 and this one in the same branch.

File: conf/modules/ins_ekf2.xml [hunk 1]
```diff
@@ -6,6 +6,14 @@
       simple INS and AHRS using EKF2 from PX4
     </description>
   </doc>
+  <settings>
+	<dl_settings NAME="INS">
+      <!-- Optical flow calculations parameters -->
```
Comment (gautierhattenberger) [inline conf/modules/ins_ekf2.xml]: wrong comment

**Comment by fvantienen** [comment]
I have rebased this one and update pprzlink. But had to add also #2502 to prevent a compile error with the sdlogger filename added to the message.
Sorry for this mess...

File: conf/modules/ins_ekf2.xml (hunk 1 above)
Comment (fvantienen) [reply conf/modules/ins_ekf2.xml]: Resolved

**Pull request merged**